.drone.yml

```
@@ -11,7 +11,7 @@ steps:
 - name: lint bash
   image: koalaman/shellcheck-alpine
   commands:
-    - shellcheck *.sh */*.sh
+    - shellcheck *.sh */*.sh */*/*.sh
```

Could this just be **/*.sh?

---

I'm not sure? `globstar` is only available in some versions of Bash, and in more recent versions is disabled by default. We could do this I guess:
```suggestion
     - shopt -s globstar
    - shellcheck **/*.sh
```

---

Was more of a can it do it out the box question. Not overly fussed...